src/swap/components/SwapProvider.tsx

```diff
@@ -231,22 +238,24 @@ export function SwapProvider({
     config,
     from.amount,
     from.token,
+    maxSlippage,
     sendTransactionAsync,
     to.token,
     useAggregator,
-    experimental.maxSlippage,
   ]);
 
   const value = useValue({
     error,
     from,
-    loading,
     handleAmountChange,
     handleToggle,
     handleSubmit,
-    lifeCycleStatus,
     isTransactionPending,
+    lifeCycleStatus,
+    loading,
+    maxSlippage,
```

Comment:
Not a fan of adding more things in the Context. As adding more things in the context, shows weakness in the data-flow of information in the architecture.

Let me read what's going on.

Reply:
Agreed that we should be careful about adding info in the Context but I think it makes sense with maxSlippage